src/entities/entity.h

```diff
@@ -0,0 +1,82 @@
+/***
+ * Maracatronics Robotics
+ * Federal University of Pernambuco (UFPE) at Recife
+ * http://www.maracatronics.com/
+ *
+ * This file is part of Armorial project.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ***/
+
+#ifndef ENTITY_H
+#define ENTITY_H
+
+#include <QThread>
+#include <QMutex>
+
+#include <src/utils/timer/timer.h>
+
+enum EntityType {
```

Comment:
This is an enumerate to define the types of objects that will be entities in our software: basically players, the vision, coach and actuator modules.